pkg/gadgets/dns/gadget.go

```diff
@@ -219,7 +214,7 @@ func (t *Trace) Start(trace *gadgetv1alpha1.Trace) {
 	)
 
 	for _, c := range existingContainers {
-		err := attachContainerFunc(c.Namespace, c.Podname, c.Pid)
+		err := attachContainerFunc(c)
 		if err != nil {
```

Comment:
This warning will never be printed because `attachContainerFunc` always return nil.